images/netcat-openbsd/configs/latest.apko.yaml

```diff
@@ -0,0 +1,26 @@
+contents:
+  repositories:
+    - https://dl-cdn.alpinelinux.org/alpine/edge/main
+    - https://dl-cdn.alpinelinux.org/alpine/edge/community
+  packages:
+    - ca-certificates-bundle
+    - busybox
+    - netcat-openbsd
```

Comment:
i think we are trying to prevent new alpine-based images in this repo.

Maybe we can look into packaging netcat in wolfi

Reply:
lemme try adding netcat-openbsd PR at wolfi repo